girder_annotation/girder_large_image_annotation/web_client/templates/annotationListWidget.pug

```diff
@@ -5,21 +5,12 @@
     if creationAccess
       a.g-annotation-upload(title='Upload annotation')
         i.icon-upload
-    if annotations.length
-      a.g-annotation-download(href=`${apiRoot}/annotation/item/${item.id}`, title='Download annotations', download=`${item.get('name')}_annotations.json`)
-        i.icon-download
-    if accessLevel >= AccessType.ADMIN && annotations.length
-      a.g-annotation-permissions(title='Adjust permissions')
-        i.icon-lock
-      a.g-annotation-delete(title='Delete')
-        i.icon-cancel
 
 if annotations.length
   table.g-annotation-list.table.table-hover.table-condensed
     thead
-      //
-        th.g-annotation-select
-          input.g-select-all(type='checkbox')
+      th.g-annotation-select
+        input(type='checkbox' title='Select all annotations for bulk actions' class='pr-2')
```

Comment:
I think linting will demand this to be `input.pr-2(type='checkbox', title='Select all annotations for bulk actions')`